Gemfile.lock

```diff
@@ -748,9 +748,6 @@ GEM
       addressable (>= 2.3.6)
       crack (>= 0.3.2)
       hashdiff
-    websocket-driver (0.7.0)
```

Comment:
I don't understand this :thinking: . We must have missed it when changing the Gemfile.lock in another PR.